tests/lib/external/snapd-testing-tools/remote/remote.wait-for

```diff
@@ -8,8 +8,8 @@ DEFAULT_WAIT_FOR_SNAP_COMMAND_ATTEMPTS=200
 DEFAULT_WAIT_FOR_SNAP_COMMAND_WAIT=1
 DEFAULT_WAIT_FOR_DEV_INIT_ATTEMPTS=60
 DEFAULT_WAIT_FOR_DEV_INIT_WAIT=1
-DEFAULT_WAIT_FOR_REBOOT_ATTEMPTS=60
-DEFAULT_WAIT_FOR_REBOOT_WAIT=1
+DEFAULT_WAIT_FOR_REBOOT_ATTEMPTS=120
```

Comment:
These values just feel incredibly arbitrary and I would love to see a comment explaining why these numbers were selected

Reply:
well, you are right, those numbers are arbitrary, they were selected trying to match with most of the wait times in the tests and also trying to follow common sense.

Reply:
It's completely fine they are arbitrary, but just a comment saying what you just told me would be cool